src/handlers/external_command_handler.rs

```diff
@@ -1,163 +0,0 @@
-use super::{command_handler, focus_handler, goto_tag_handler, Command, Manager};
-use crate::config::Config;
-use crate::config::ThemeLoader;
-use crate::config::ThemeSetting;
-use crate::state::State;
-use crate::utils::command_pipe::ExternalCommand;
-use crate::utils::window_updater::update_windows;
-use std::sync::Arc;
-
-pub fn process(
-    manager: &mut Manager,
-    state: &impl State,
-    config: &impl Config,
-    theme_loader: &impl ThemeLoader,
-    command: ExternalCommand,
-) -> bool {
-    let needs_redraw = process_work(manager, state, config, theme_loader, command);
-    if needs_redraw {
-        update_windows(manager);
-    }
-    needs_redraw
```

Comment:
Ohhh that code? @AethanFoot Ok will make a PR quickly oops

Reply:
Ye, idk it will be better to put it in the event loop, and move the one out of the display event handler